application/config/config.go

```diff
@@ -346,8 +347,17 @@ func (c *Config) LogPath() string {
 	defer c.m.Unlock()
 	return c.logPath
 }
-func (c *Config) SnykApi() string                        { return c.snykApiUrl }
-func (c *Config) SnykCodeApi() string                    { return c.snykCodeApiUrl }
+func (c *Config) SnykApi() string     { return c.snykApiUrl }
+func (c *Config) SnykCodeApi() string { return c.snykCodeApiUrl }
+func (c *Config) SnykUi() string {
+	parsedUrl, err := url.Parse(c.snykApiUrl)
+	if err != nil || !parsedUrl.IsAbs() {
+		return DefaultSnykUiUrl
+	}
+	host := parsedUrl.Host
+	appHostname := strings.Replace(host, "api", "app", 1)
+	return fmt.Sprintf("%s://%s", parsedUrl.Scheme, appHostname)
+}
```

Comment:
Shall we add a unit test for this logic? Maybe we could check if `c.snykApiUrl` is not a valid or absolute URL and the happy paths? `TestSnykCodeApi` seems to be covering a few interesting cases that we could have a look?

Reply:
I added a couple of tests similar to the `TestSnykCodeApi` tests